Vagrantfile

```diff
@@ -0,0 +1,48 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/xenial64"
+
+  config.vm.provision "shell", inline: <<-SHELL
+
+    echo "Provisioning Start"
+    export DEBIAN_FRONTEND=noninteractive
+    apt-get -qq update && apt-get -qq dist-upgrade
+
+    echo "Installing MySql"
+    sudo -E apt-get -qq -y install mysql-server
+    /etc/init.d/mysql stop
+    mysqld --skip-grant-tables &
+    mysql -u root -e "UPDATE user SET Password=PASSWORD('pass123') WHERE User='root'; FLUSH PRIVILEGES; exit;"
```

Comment:
The no interactive install wasn't working, so I resorted to resetting the password after the installation process set it to something (not really sure what, it should have left it blank), but either way the goal of this is to allow people to have a complete working dev environment with one command, and having them set the password will mean that they also have to run the migrations themselves.